netty/shaded/src/testShadow/java/io/grpc/netty/shaded/ShadingTest.java

```diff
@@ -69,6 +74,19 @@ public void noNormalNetty() throws Exception {
     Class.forName("io.grpc.netty.NettyServerBuilder");
   }
 
+  /** Verify that resources under META-INF/native-image reference shaded class names. */
+  @Test
+  public void nettyResourcesUpdated() throws IOException {
+    InputStream inputStream = NettyChannelBuilder.class.getClassLoader()
+        .getResourceAsStream("META-INF/native-image/io.netty/transport/reflection-config.json");
```

Comment:
We really ought to rename the files as well (the "io.netty" folder), to avoid them colliding with Netty's files. But we can do that as a follow-up.